automaxlair/matchup_scoring.py

```diff
@@ -225,7 +224,7 @@ def calculate_move_score(attacker: Pokemon, move_index: int, defender: Pokemon,
                 pass
         else:
             received_damage += 0.25 * calculate_damage(defender, i, attacker,
-                multiple_targets=True) / (2 if attacker.dynamax else 1)
+                multiple_targets=False) / (2 if attacker.dynamax else 1)

```

Comment:
Oh I think that multiple_targets needs to be True for both this and the next function call. Is that right?

Reply:
It's in the else of spread, so i think it should be False

Reply:
Oh true. I'm not sure it will make a difference if the move isn't a spread move.